components/form/checkbox/src/index.js

```diff
@@ -0,0 +1,74 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+const FormCheckbox = ({checked, className, errorMessage, label, name, onChange, svgIcons}) => (
+  <div className={cx('sui-FormCheckbox', className)}>
+    <label className='sui-FormCheckbox-container'>
+      <input
+        checked={checked}
+        className={cx('sui-FormCheckbox-input', {'is-checked': checked})}
+        name={name}
+        onChange={onChange}
+        type='checkbox' />
+      {checked ? svgIcons.checked : svgIcons.unchecked}
+      <span className='sui-FormCheckbox-label'>{label}</span>
+    </label>
+    {errorMessage &&
+    <div className='sui-FormCheckbox-errorMessage'>
+      <span className='sui-FormCheckbox-errorMessageLabel'>{errorMessage}</span>
+    </div>
+    }
+  </div>
+)
+
+FormCheckbox.displayName = 'FormCheckbox'
+
+FormCheckbox.propTypes = {
+  /**
+   * CSS classname to apply to component's container.
+   */
+  className: PropTypes.string,
+  /**
+   * Boolean attribute that specifies if the input should be displayed as selected or not.
+   */
+  checked: PropTypes.bool,
+  /**
+   * Text to display as error message below the checkbox label.
+   */
+  errorMessage: PropTypes.string,
+  /**
+   * Text to display as a description message on the right side of the chechbox input.
+   */
+  label: PropTypes.node,
+  /**
+   * Specifies the name of the checkbox component.
+   */
+  name: PropTypes.string.isRequired,
```

Comment:
Is there any special reason this is required?

Reply:
Not really, I wanted to have an unique id/name to be used in parent's form, but using the form's state is enough. I will remove this.